### PR TITLE
misc changes

### PR DESCRIPTION
**Pull request by tosch42**
this updates the lchat manpage and makes some other small changes. I'll contribute a manpage for slackline later on.

### CODE REVIEW

File: slackline.c [hunk 1]
```diff
@@ -158,20 +158,20 @@ static void
...
 	switch (key) {
-	case 27:	/* Escape */
+	case ESC_KEY:
 		sl->esc = ESC;
 		break;
-	case 21:
+	case CTRL_U:
 		sl_reset(sl);
 		break;
-	case 23: /* ctrl+w -- erase previous word */
```
Comment (younix) [inline slackline.c:167]: please keep the "erase previous word" comment.

File: slackline_emacs.c [hunk 2]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
 	char tmp;
 
 	switch (key) {
-	case 27:	/* Escape */
+	case ESC_KEY:
 		sl->esc = ESC;
 		break;
-	case 1: /* ctrl+a -- start of line */
```
Comment (younix) [inline slackline_emacs.c:17]: please keep the "start of line" comment

File: slackline_emacs.c [hunk 3]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
+	case CTRL_A:
 		sl_move(sl, HOME);
 		break;
-	case 2: /* ctrl+b -- previous char */
+	case CTRL_B:
 		sl_move(sl, LEFT);
 		break;
-	case 4: /* ctrl+d -- delete char in front of the cursor or exit */
```
Comment (younix) [inline slackline_emacs.c:23]: please keep the "delete char in front of the cursor or exit" comment.

File: slackline_emacs.c [hunk 4]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
 	switch (key) {
-	case 27:	/* Escape */
+	case ESC_KEY:
 		sl->esc = ESC;
 		break;
-	case 1: /* ctrl+a -- start of line */
+	case CTRL_A:
 		sl_move(sl, HOME);
 		break;
-	case 2: /* ctrl+b -- previous char */
```
Comment (younix) [inline slackline_emacs.c:20]: please keep the "previous char" comment.

File: slackline_emacs.c [hunk 5]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
-	case 4: /* ctrl+d -- delete char in front of the cursor or exit */
+	case CTRL_D:
 		if (sl->rcur < sl->rlen) {
 			sl_move(sl, RIGHT);
 			sl_backspace(sl);
 		} else {
 			exit(EXIT_SUCCESS);
 		}
 		break;
-	case 5: /* ctrl+e -- end of line */
```
Comment (younix) [inline slackline_emacs.c:31]: please keep the "end of line" comment.

File: slackline_emacs.c [hunk 6]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
+	case CTRL_E:
 		sl_move(sl, END);
 		break;
-	case 6: /* ctrl+f -- next char */
```
Comment (younix) [inline slackline_emacs.c:34]: please keep the "next char" comment.

File: slackline_emacs.c [hunk 7]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
+	case CTRL_F:
 		sl_move(sl, RIGHT);
 		break;
-	case 11: /* ctrl+k -- delete line from cursor to end */
```
Comment (younix) [inline slackline_emacs.c:37]: please keep the "delete line from cursor to end" comment

File: slackline_emacs.c [hunk 8]
```diff
@@ -11,36 +11,36 @@ sl_emacs(struct slackline *sl, int key)
...
+	case CTRL_K:
 		for (int i = sl->rlen - sl->rcur; i > 0; --i) {
 			sl_move(sl, RIGHT);
 			sl_backspace(sl);
 		}
 		break;
-	case 20: /* ctrl+t -- swap last two chars */
```
Comment (younix) [inline slackline_emacs.c:43]: please keep the "swap last two chars" comment.

**Review by younix — COMMENTED** [review]
good work. just keep some of the comments i've marked.
someone who is not familiar with emacs will need them,
to understand the code peaces below.

**Comment by tosch42** [comment]
makes sense, done in the last commit

**Pull request merged**